File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// EmailValidation SDK exists test

require_once __DIR__ . '/../emailvalidation_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = EmailValidationSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# EmailValidation SDK exists test

require "minitest/autorun"
require_relative "../EmailValidation_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = EmailValidationSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
